### DISCUSSION TITLE
Power Behind the Throne Formatting

### DISCUSSION THREAD

**Discussion by ericcberg** [moo-man/WFRP4e-FoundryVTT#1583] [General]
Wondering why the chapters in Power Behind the Throne are folders full of individual pages rather than single pages with sub-pages like the previous two modules in Enemy Within? When reading through it, it's a lot less convenient to navigate. Also, all of the links to the previous modules don't appear to work. Is that because it is set up for version 9 instead of 10?

**Comment by moo-man** [comment]
V9 -> V10 Journal conversions take time, and I'm performing them in release order. The most recent conversion completed was Death on the Reik. Regarding links, see https://github.com/moo-man/WFRP4e-FoundryVTT/issues/1278

**Reply by ericcberg** [reply]
Awesome! These are amazing, btw. You do fantastic work.

**Reply by moo-man** [reply]
Thanks! Glad you like it
